codebuild/build_image.yaml

```diff
@@ -1,5 +1,8 @@
 version: 0.2      
 phases:
+  pre_build:
```

Comment:
do we also need to add it to unit test, build_canary and deploy_image as well ?

deploy_image also fails sometimes so yes for that. not sure about the rest